src/ios/SOSPicker.m

```diff
@@ -280,5 +281,13 @@ -(void)assetsPickerControllerDidCancel:(GMImagePickerController *)picker
    NSLog(@"GMImagePicker: User pressed cancel button");
 }
 
+- (void) closeImagePicker:(CDVInvokedUrlCommand *)command {
+    CDVPluginResult* pluginResult = nil;
+    NSArray* emptyArray = [NSArray array];
+    pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsArray:emptyArray];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+    [_imagePicker.presentingViewController dismissViewControllerAnimated:YES completion:nil];
```

Comment:
```suggestion
    [_imagePicker.presentingViewController dismissViewControllerAnimated:YES completion:nil];
```
Where are we getting the variable `_imagePicker`?

Reply:
line 86

Reply:
use `self` instead of `_`